lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -169,10 +148,33 @@
   suspend_processes:
     description:
       - A list of scaling processes to suspend.
-    required: False
     default: []
     choices: ['Launch', 'Terminate', 'HealthCheck', 'ReplaceUnhealthy', 'AZRebalance', 'AlarmNotification', 'ScheduledActions', 'AddToLoadBalancer']
     version_added: "2.3"
+  metrics_collection:
```

Comment:
Ok, so it is weird that we're backporting modules parameter improvements in v2.5 ?
Shouldn't this already be in v2.5 ??